easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.2-2023b.yml

```
@@ -15,3 +15,7 @@ easyconfigs:
       options:
         # see https://github.com/easybuilders/easybuild-easyconfigs/pull/21034
         from-commit: 76e7fc6657bab64bfbec826540a3a8f0040258f2
+  - STAR-2.7.11b-GCC-13.2.0.eb:
+      options:
+        #see https://github.com/easybuilders/easybuild-easyconfigs/pull/21200
+        from-commit: cd8b1ce595195f132125d9b25a98a9efe702c72a
```

I've pushed in some more updates into the PR, mostly to make the changes consistent across recent STAR easyconfigs:
```suggestion
        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/21200
        from-commit: 765ba900daf5953e306c4dad896febe52fdd6c00
```